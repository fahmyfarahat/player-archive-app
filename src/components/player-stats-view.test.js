import React from "react";
import { shallow } from "enzyme";
import PlayerStatsView from "./player-stats-view";
import toJson from "enzyme-to-json";

describe("PlayerStatsView - general", () => {
  let wrapper;
  beforeEach(()=> {
    const props = {
      title: "title",
      statsToShow: ["blocks"],
      stats: {
        blocks: "blocks"
      }
    }
    wrapper = shallow(<PlayerStatsView {...props}/>);
  });
  
  it("should render without crashing", () => {
    expect(wrapper).toHaveLength(1);
  });
  
  it("should match Snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
