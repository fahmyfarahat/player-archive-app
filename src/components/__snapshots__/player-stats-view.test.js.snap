// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerStatsView - general should match Snapshot 1`] = `
<div
  className="player-stats"
>
  <div
    className="player-stats_head"
  >
    <h3>
      title
    </h3>
  </div>
  <div
    className="player-stats_body"
  >
    <div
      className="player-stats_row"
      key="blocks"
    >
      <span
        className="text-capitalize"
      >
        blocks
      </span>
      <span
        className="font-weight-bold"
      >
        blocks
      </span>
    </div>
  </div>
</div>
`;
